src/zproxy.c

```diff
@@ -191,12 +203,23 @@ s_self_configure (self_t *self, zsock_t **sock_p, zmsg_t *request, proxy_socket
     assert (type_name);
     char *endpoints = zmsg_popstr (request);
     assert (endpoints);
+
+    zproxy_heartbeat_t zproxy_heartbeat = { 0, };
+    char *heartbeat = zmsg_popstr (request);
+    if (heartbeat && strncmp (heartbeat, "HEARTBEAT", strlen ("HEARTBEAT")) == 0) {
```

Comment:
don't cram it like this, use separate message parts for each parameter instead

Reply:
Thanks! I changed to multiple message parts!